lib/v2/screens/explore_screens/invite/invite_screen.dart

```diff
@@ -22,9 +24,35 @@ class InviteScreen extends StatelessWidget {
       child: Scaffold(
         appBar: AppBar(title: Text('Invite', style: Theme.of(context).textTheme.headline7)),
         body: BlocConsumer<InviteBloc, InviteState>(
-          listenWhen: (_, current) => current.showInviteLinkDialog,
+          listenWhen: (_, current) => current.pageCommand != null,
           listener: (context, state) {
-            // Here will go a invite link dialog
+            BlocProvider.of<InviteBloc>(context).add(const ClearInviteScreenPageCommand());
+            if (state.pageCommand is ShowInviteLinkDialog) {
+              showDialog<void>(
+                context: context,
+                barrierDismissible: false,
+                builder: (_) {
+                  return BlocProvider.value(
+                    value: BlocProvider.of<InviteBloc>(context),
+                    child: const InviteLinkDialog(),
+                  );
+                },
+              );
+            }
+            if (state.pageCommand is ShowTransactionFailSnackBar) {
+              Navigator.of(context).pop();
+              ScaffoldMessenger.of(context).showSnackBar(
+                SnackBar(
+                  behavior: SnackBarBehavior.floating,
+                  backgroundColor: AppColors.grey,
```

Comment:
Whats the color if we dont set it?

Reply:
white

Reply:
We can define a default color int he globla them using a SnackBarThemeData. 
The question is what would be that default color?